_data/changelogs/side-navigation.yml

```diff
@@ -19,4 +19,49 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
```

Comment:
I removed this line because I was duplicating the settings and this line seemed unnecessary